symengine/cwrapper.cpp

```diff
@@ -186,6 +187,11 @@ void integer_set_str(basic s, char *c)
     s->m = SymEngine::integer(integer_class(c));
 }
 
+void real_double_set_str(basic s, char *c)
```

Comment:
Instead of this. `integer` one has several types of input. In this case though, a double is enough.